__tests__/PaginationBoxView-test.js

```diff
@@ -861,18 +866,19 @@ describe('Test custom props', () => {
 
     it('should update forcePage and hence selected page when forcePage value is changed', () => {
       const node = document.createElement('div');
-      const pagination1 = ReactDOM.render(
-        <PaginationBoxView forcePage={2} />,
-        node
-      );
-      const pagination2 = ReactDOM.render(
-        <PaginationBoxView forcePage={3} />,
-        node
+      ReactDOM.render(<PaginationBoxView forcePage={2} />, node);
+      const pagination = React.createRef();
+      ReactDOM.render(
+        <PaginationBoxView ref={pagination} forcePage={3} />,
+        node,
+        () => {
+          expect(
+            ReactDOM.findDOMNode(pagination.current).querySelector(
+              '.selected a'
+            ).textContent
+          ).toBe('3');
+        }
```

Comment:
This is the test that needed major modification. Now instead of using the return of `ReactDOM.render` (which is not recommended) I'm storing a ref to the component and then using that ref in the render callback (which fires once the component is mounted)

I also have to change the expected output from `4` to `3` which based on the `forcePage` attribute I believe is correct and the `4` was originally put there due to a side effect of using the .render reference